specification/maps/data-plane/Route/preview/2023-10-01-preview/route.json

```diff
@@ -312,8 +312,8 @@
               "modelAsString": true,
               "values": [
                 {
-                  "value": "highways",
-                  "description": "Avoids the use of highways in the route. `Note`: Only supported for driving travelMode."
+                  "value": "limitedAccessHighway",
+                  "description": "Avoids the use of limited access highway in the route. `Note`: Only supported for driving and truck travelMode."
                 },
```

Comment:
@koyasu221b We don't need to mention only supported for driving and truck if it's supported by both travel modes like ferries and toll road and it is implied that avoid highways would not be applicable to walking.